src/core/model.js

```diff
@@ -241,6 +246,9 @@ export default class Model extends Object3D {
    */
   /* eslint-disable max-statements */
   render(uniforms = {}) {
+    if (window.__SEER_INITIALIZED__) {
+      addModel(this);
```

Comment:
Hmm, maybe yes, maybe some models are overwriting it in their constructors rather than passing it in as a parameter.

If you make an ES6 Map you could key on objects instead of ids.

It wouldn't work in deck.gl but it would work here.